auth0/lock
# Added unhandled authorization error

### Changes

We found an issue with the "last time you logged in with" screen where the silent auth fails because the inactivity timeout/required login after time limit was reached. As if someone went on vacation, came back, and tried to sso back in. The 'login_required' error gets fired because the user needs to re-authenticate but was not emitted as an authorization error. Without this change, there is no way to handle this scenario as it fails to fire lock.on('authorization_error') event.

### References

Support Ticket #00426103

### Testing

Repro steps:
1. Have a lock with rememberLastLogin set to true
2. Go to Tenant > Advanced > Log In Sesion Management
3. Set the inactivity timeout to 1 minute
4. Login normally with the lock
5. Wait a minute (note: your lock will need to be triggered with .show() to display the "last time you logged in with" screen)
6. Remember last login will fire and when you click on it, the error text "WE'RE SORRY, SOMETHING WENT WRONG WHEN ATTEMPTING TO LOG IN." will appear and no authorization_error is fired.


### Checklist

* [x] All code quality tools/guidelines have been run/followed
* [x] All relevant assets have been compiled
